src/components/SkeletonPage/SkeletonPage.scss

```diff
@@ -32,6 +32,43 @@ $action-text-spacing: rem(24px);
   margin-bottom: (-1 * spacing(extra-tight));
 }
 
+.TitleAndPrimaryAction {
```

Comment:
I'm not confident about the name of this class. What about `HeaderActions`?

Reply:
I think that could lead to confusion since this grouping does not include Secondary Actions.

Reply:
`PrimaryActions`?

Reply:
For posterity, Liz and I chatted about this and she pointed out that his naming convention is used in `Page` `Header`. She's going to open a separate issue to address this.